utils/testgen.py

```diff
@@ -181,7 +181,8 @@ def fixture_filter(metafunc, argnames, argvalues):
     keep_index = [e[0] for e in enumerate(argnames) if e[1] in metafunc.fixturenames]
```

Comment:
future todo? we should use index, name here, and generate a itemgetter to be applied to the list instead of just a normal list expression function